routes/contact.js

```diff
@@ -1,6 +1,35 @@
 const express = require('express')
+const EmailManager = require('../utils/email')
 const router = express.Router()
 
-router.get('/', (req, res) => res.send('Contact API.'))
+const confirmation = {
+    subject: 'Confirmation from MAI Youth',
+    text: 'Thank you for contacting the MAI Youth team.\n\nThis is to confirm we got your query and we\'ll respond to it as soon as possible.\n\nKind regards,\nMAI Youth Team',
+}
+
+router.post('/send', (req, res) => {
+    const { email, subject, text } = req.body
+    const newSubject = `CF: ${subject}`
```

Comment:
> What does `CF: ${subject}` do?

It substitutes variables by setting `${VAR_NAME}`. [This](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) describes it well.

@mahmoudal99